.github/workflows/autobuild.yml

```diff
@@ -412,34 +415,17 @@ jobs:
         with:
           product-path:             deploy/${{ steps.get-artifacts.outputs.artifact_1 }}
 
-      - name:                       Upload Artifact 1 to Release
-        # Every build job has at least one artifact. Therefore, no `if artifact_1` condition here.
-        # If the artifact is missing, this should fail.
+      - name:                       Deploy Artifacts to Release
         if:                         needs.create_release.outputs.publish_to_release == 'true'
-        id:                         upload-release-asset1
-        uses:                       actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url:               ${{ needs.create_release.outputs.upload_url }}
-          asset_path:               deploy/${{ steps.get-artifacts.outputs.artifact_1 }}
-          asset_name:               ${{ steps.get-artifacts.outputs.artifact_1 }}
-          asset_content_type:       application/octet-stream
-
-      - name:                       Upload Artifact 2 to Release
-        if:                         >-
-                                    steps.get-artifacts.outputs.artifact_2 != '' &&
-                                    needs.create_release.outputs.publish_to_release == 'true'
-        id:                         upload-release-asset2
-        uses:                       actions/upload-release-asset@v1
+        id:                         upload-release-assets
+        run:                        >
+          gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" "deploy/${{ steps.get-artifacts.outputs.artifact_2 }}" --clobber
+          ||
+          gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" --clobber
```

Comment:
🤦 I just thought... why not test for the artifact presence before trying to upload it?  (It's two separate operations under the covers and this avoids succeeding to upload artifact_1, partly uploading artifact_2 then failing to upload the already existing artifact_1:
```yaml
run: -
  gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" --clobber
  [[ -f "deploy/${{ steps.get-artifacts.outputs.artifact_2 }}" ]] && gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_2 }}" --clobber
```

Reply:
Because Powershell doesn't support this syntax as far as I know.

Reply:
OK, the yaml's just fed to host shell, of course.  But this is the artifact deploy, not the build, isn't it..?  Does it run on top of PowerShell?

Reply:
I think so. Or CMD but I'm quite sure it's not bash.

Reply:
Looks like it must be `cmd`, then - Powershell doesn't like the syntax when I try it.  (`echo hello || echo goodbye` for example.)

Reply:
Anyway, at the moment, this works out as
> Upload the first file
> If that was okay (script still running), try uploading the second file but don't worry if that fails

So
```
run: -
  gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" --clobber
  gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_2 }}" --clobber || echo "Ignoring failed artifact_2 upload"
```

Reply:
> Powershell doesn't like the syntax when I try it. (echo hello || echo goodbye for example.)

Strange. I read that it was the other way around. I'll fire up a test build.

Reply:
See: https://stackoverflow.com/questions/2416662/what-are-the-powershell-equivalents-of-bashs-and-operators/41816341#41816341 which I used as reference

Reply:
And that's the relevant issue: https://github.com/cli/cli/issues/5099

Reply:
@pljones see https://github.com/ann0see/jamulus/releases/tag/r3_9_1devTestPlainGHRelease - so it works with the current hacky syntax.

Reply:
Given all the preceeding steps (e.g. use of `apt-get` to install `gh`), I'd expect it to be on bash...  Things like `set -eu` aren't very portable :).

Reply:
Yes. But we have a conditional there. I wanted to avoid them as it's a simple invocation of gh which *should* behave the same on all OS

Reply:
We've got platform-dependent code even in `Jamulus.pro`, which isn't nice.  (Means building for Android on Windows doesn't work -- there's a `wc -l` call...)

But as I say, here we might as well do the first artifact, then, if that's worked, do the second artifact, then, if that fails, continue.  As that appears to be the only option for checking whether artifact two is present in the same step.

Reply:
Not the only one. We could also augment the building scripts as in the initial state of this PR -> Each OS has a custom "upload artifact to release" step comparable to the build.

Reply:
@pljones I've now fixed the PR for the repo too. What do you think?

Reply:
> Anyway, at the moment, this works out as
> 
> > Upload the first file
> 
> > If that was okay (script still running), try uploading the second file but don't worry if that fails
> 
> 
> 
> So
> 
> ```
> 
> run: -
> 
>   gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_1 }}" --clobber
> 
>   gh release upload "${{ needs.create_release.outputs.tag_name }}" "deploy/${{ steps.get-artifacts.outputs.artifact_2 }}" --clobber || echo "Ignoring failed artifact_2 upload"
> 
> ```

Probably that's more readable, true. Hopefully PowerShell supports that